### PR TITLE
Support get applied clusterSubscribeRate

### PR DESCRIPTION
**Pull request by 315157973**
Master Issue: #9216

### Modifications
1. Add applied API for topic-level
2. Add remove for namespace-level

### Verifying this change
Verify applied API and CMD

### CODE REVIEW

**Review by codelipenghui — APPROVED** [review]

**Review by sijie — APPROVED** [review]

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java [hunk 1]
```diff
@@ -1256,16 +1256,27 @@ protected void internalSetSubscribeRate(SubscribeRate subscribeRate) {
...
         validateNamespacePolicyOperation(namespaceName, PolicyName.RATE, PolicyOperation.READ);
         Policies policies = getNamespacePolicies(namespaceName);
-        SubscribeRate subscribeRate = policies.clusterSubscribeRate.get(pulsar().getConfiguration().getClusterName());
-        if (subscribeRate != null) {
-            return subscribeRate;
-        } else {
-            throw new RestException(Status.NOT_FOUND,
-                    "Subscribe-rate is not configured for cluster " + pulsar().getConfiguration().getClusterName());
-        }
+        return policies.clusterSubscribeRate.get(pulsar().getConfiguration().getClusterName());
```
Comment (lhotari) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:1279]: @codelipenghui @315157973 This introduces more thread safety issues, similar to what is reported in #9711 . That's the reason why I think it's better to address the thread safety issues before introducing more problems.

Comment (lhotari) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:1279]: I referring to the unsafe mutation on line 1265 `policies.clusterSubscribeRate.remove(pulsar().getConfiguration().getClusterName())` and the unsafe access on line 1279 `policies.clusterSubscribeRate.get(pulsar().getConfiguration().getClusterName())`. 
Perhaps I'm missing something here. Do you consider this thread safe?

Comment (315157973) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:1279]: Thread safety issues will be revised uniformly, including this PR

**Comment by 315157973** [comment]
Since the thread safety issue is solved separately, the changes here are not involved, please take a look again @lhotari

**Pull request merged**